forte/solvers/mcscf.py

```diff
@@ -0,0 +1,85 @@
+# import psi4
```

Comment:
Why is this file here but commented out?

Reply:
Deleted.